### ISSUE TITLE
Canon 80D not working with libgphoto2 2.5.13 (was "Canon 80D working on one mac, not in another")

### ISSUE THREAD

**Issue by MarcSallent** [gphoto/gphoto2#71]
Hello,

I have a macbook pro and an imac, both with OS X El Capitan. Testing with a Canon 80D, it works in the macbook pro but not in the imac. 
Some commands like `gphoto2 --auto-detect` always work, but most other commands not work, like `gphoto2 --capture-image` or `gphoto2 --capture-movie` don't.

The error is always the same:                         
>*** Error ***              
>PTP I/O Error
>
>*** Error ***              
>An error occurred in the io-library ('I/O problem'): No error description available
>*** Error (-7: 'I/O problem') ***       

This is the result of `gphoto2 -v` in the imac (not working).
>gphoto2         2.5.11         clang, popt(m), no exif, no cdk, no aa, jpeg, readline
>libgphoto2      2.5.13         all camlibs, clang, ltdl, no EXIF
>libgphoto2_port 0.12.0         clang, ltdl, USB, serial without locking

The only difference is that the mbp (working) uses **libgphoto2 2.5.12**. Both installed via **brew**

I'm attaching the error log [my-logfile.txt](https://github.com/gphoto/gphoto2/files/1042338/my-logfile.txt)

Thanks!

**Comment by msmeissn** [comment]
The error in the logfile is a timeout during the first communication...

is the Mac PTPCamera process terminated?

**Comment by MarcSallent** [comment]
Yes, with `killall PTPCamera`

**Comment by MarcSallent** [comment]
I have been able to further debug this problem:

 - My Canon EOS 80D works correctly with libgphoto2 2.5.**12**
 - The new libgphoto2 2.5.**13** not only does not work correctly with this camera (`gphoto2 --capture-image-and-download` and many other commands won't work)
 - _Sometimes_, the new libgphoto2 2.5.**13**  _temporarily corrupts the camera_ (cannot find a better way to say it) and it does let it work correctly with the old libgphoto2 2.5.**12** in another computer unless I turn the camera off and wait a few minutes.

My solution has been to install libgphoto2 2.5.**12** in all my systems. I have tested this behavior and solution with OS X 10.11.6 and two different Raspberry Pis

**Comment by msmeissn** [comment]
could you also test current git?

there was 1 weird change in 2.5.13 which lead to EOS camera crashes which i have now reverted in git

**Comment by MarcSallent** [comment]
I installed it in a Raspberry Pi 3 using [gonzalo/gphoto2-updater](https://github.com/gonzalo/gphoto2-updater) script, choosing the _development version_. Many things are working again, like `gphoto2 --capture-image`, downloading files, deleting files, etc. but I cannot record movies like I can with the prev version of libgphoto2:

`gphoto2 --set-config viewfinder=1 --set-config movierecordtarget=Card --wait-event=10s --set-config movierecordtarget=None --wait-event-and-download=2s`

This is the error that I get:
```
*** Error ***              
The property 'Recording Destination' / 0xd1b8 was not set (0x2019: PTP Device Busy).
Waiting for 2 seconds for events from camera. Press Ctrl-C to abort.

```
And this is the logfile:
[log.txt](https://github.com/gphoto/gphoto2/files/1044911/log.txt)


`gphoto2 -v` says:

```
gphoto2 2.5.11.1
...
This version of gphoto2 is using the following software versions and options:
gphoto2         2.5.11.1       gcc, popt(m), exif, no cdk, no aa, jpeg, no readline
libgphoto2      2.5.13.1       all camlibs, gcc, ltdl, EXIF
libgphoto2_port 0.12.0         gcc, ltdl, USB, serial without locking

```

**Comment by samidalati** [comment]
Hi, 
is this bug with Canon 80D fixed?

**Comment by MarcSallent** [comment]
Unfortunately I don't have access to the Canon 80D, I cannot test it with libgphoto2 2.5.14 :\

**Comment by msmeissn** [comment]
this is likelyu fixed in 2.5.14. if not, please reopen this issue
